README.md

```diff
@@ -131,9 +62,38 @@ The CrowdRender documentation can be found [here](https://www.crowd-render.com/d
 - `-e persistent true|false`: set the container to be reigstered as a persistent node; default: `false`
 - `--hostname`: change hostname of the docker container (this is the name that will appear on the CR clients list)
 
+## `CR_VERSION` Table
+
+| CR_VERSION           | Crowdrender version              | Blender version |
+|:--------------------:|:--------------------------------:|:---------------:|
+| `latest`             | latest available for the account | `2.80 - 3.2`    |
+| `cr_044_bl280`       | `0.4.4`                          | `2.80 - 3.2`    |
+| `cr_043_bl280`       | `0.4.3`                          | `2.80 - 3.2`    |
+| `cr_042_bl280`       | `0.4.2`                          | `2.80 - 3.2`    |
+| `cr_041_bl280`       | `0.4.1`                          | `2.80 - 2.93`   |
+| `cr_040_bl280`       | `0.4.0`                          | `2.80 - 2.93`   |
+| `cr_032_bl280`       | `0.3.2`                          | `2.80 - 2.93`   |
+| `cr_031_bl280`       | `0.3.1`                          | `2.80 - 2.93`   |
+| `cr_030_bl280`       | `0.3.0`                          | `2.80 - 2.93`   |
+| `cr_0210_bl280`      | `0.2.10`                         | `2.80 - 2.93`   |
+| `cr_025_bl279`       | `0.2.5`                          | `2.79b`         |
+| `cr_024_bl279`       | `0.2.4`                          | `2.79b`         |
+| `cr_023_bl280`       | `0.2.3`                          | `2.79b`         |
+| `cr_022_bl279`       | `0.2.2`                          | `2.79b`         |
+| `cr_021_bl279`       | `0.2.1`                          | `2.79b`         |
+| `cr_020_bl279`       | `0.2.0`                          | `2.79b`         |
+| `cr_018_bl279`       | `0.1.8`                          | `2.79b`         |
+| `cr_017`             | `0.1.7`                          | `2.79b`         |
+| `cr_016`             | `0.1.6`                          | `2.79b`         |
+| `cr_015`             | `0.1.5`                          | `2.79b`         |
+| `cr_014_bl279_Patch` | `0.1.4`                          | `2.79b`         |
+
+**IMPORTANT:** the latest free CR version is `0.4.1`, ***everything above is only available for supporters of our dev fund, [learn more here](https://www.crowd-render.com/crowdfunding)!***
+
 ## Extra Bits
 
-**if you have any problems or questions, you can create an [issue](https://github.com/crowdrender/cr-docker/issues) or feel free to contact us: info(at)crowdrender(dot)com(dot)au**
+**if you have any problems or questions, you can create an [issue](https://github.com/crowdrender/cr-docker/issues) or feel free to contact us: `info(at)crowdrender(dot)com(dot)au`**
```

Comment:
I'm not convinced of the value of obfuscating email addresses in order to make it harder for bots to parse since it makes it even harder for a human to parse and use. Happy either way but my preference is for clickable email addresses.

Reply:
Happy to change, though what evidence is there that this form of obsfucating isn't worth doing? Would like to make sure we're not changing this unless its backed by data. On the other hand, we do seem to have a very successful spam filter on our current mail system.

Reply:
The answer here basically elaborates on my view. It's not data in the sense of measuring spam with or without email obfuscation, but better than just my opinion on it's own. I think the ideal solution to this is to direct users to a contact form which doesn't require an email address to be exposed at all, that way it's only a single click for humans, and regular form submission bot filtering like captcha can get rid of the vast majority of spam.

Reply:
https://webmasters.stackexchange.com/questions/88043/does-it-really-help-against-spam-to-replace-the-with-at-and-similar-prac

Reply:
Agreed, captured [here](https://github.com/crowdrender/cr-docker/issues/17)